crates/internal/src/command/lightning.rs

```diff
@@ -46,10 +47,11 @@ pub async fn handle_command(
         LightningCommand::CreateInvoice {
             amount_sat,
             description,
+            expiry_sec,
```

Comment:
Let's keep naming in line with the other places

```suggestion
            expiry_secs,
```